challenges/2021_CyrilMorcrette_cloudBaseHeight/cbh_challenge_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #show cubes
</pre><p>this is quite a large display, maybe rather than show all cubes, you coiuld just print out a list of cube names, then show the "fancy" html cube display for just one cube, since they're mostly the same shape.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>